classes/plugins/PubObjectsExportPlugin.php

```diff
@@ -785,12 +785,16 @@ public function executeCLICommand($scriptName, $command, $context, $outputFile,
      */
     public function getPublishedSubmissions($submissionIds, $context)
     {
-        $submissions = array_map(function ($submissionId) {
+        $allContextSubmissionIds = Repo::submission()
```

Comment:
Not super important but `$allContextSubmissionIds` sounds like it should be for all contexts, not all submissions for a single context.

Reply:
:+1: I will then call it `allSubmissionIds` :-) Hope that that is OK...